openwisp-qa-check

```diff
@@ -60,6 +60,16 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  echo "RUNNING: Stylelint checks"
+  stylelint -f verbose $(find . -type f -name "*.css")
```

Comment:
I think this command would spit a lot of unnecessary information even when the test passes.
To avoid it, checkout the how the other functions are suppressing output on success.
Example:
```
runstylelintcheck() {
  stylelint $(find . -type f -name "*.css") &&
    echo "SUCCESS: Stylelint check successful!" ||
    {
      echoerr "ERROR: Stylelint check failed! Hint: did you forget running openwisp-qa-format?"
      FAILURE=1
    }
}
```